src/controller.rs

```diff
@@ -142,15 +143,28 @@ impl ContextData {
 pub async fn run() {
     let client: Client = Client::try_default().await.expect("Failed to create kube Client");
     let owned_api: Api<Hoprd> = Api::<Hoprd>::all(client.clone());
+    let deployment = Api::<Deployment>::all(client.clone());
+    let service = Api::<Service>::all(client.clone());
+    let service_monitor = Api::<ServiceMonitor>::all(client.clone());
+    let ingress = Api::<Ingress>::all(client.clone());
+
     let context_data: Arc<ContextData> = Arc::new(ContextData::new(client.clone()).await);
     Controller::new(owned_api, ListParams::default())
+        .owns(deployment, ListParams::default())
+        .owns(service, ListParams::default())
+        .owns(service_monitor, ListParams::default())
+        .owns(ingress, ListParams::default())
         .shutdown_on_signal()
         .run(reconciler, on_error, context_data)
         .for_each(|reconciliation_result| async move {
             match reconciliation_result {
                 Ok(_hoprd_resource) => {}
                 Err(reconciliation_err) => {
-                    eprintln!("[ERROR] Reconciliation error: {:?}", reconciliation_err)
+                    let err_string = reconciliation_err.to_string();
```

Comment:
`to_string` typically returns an `Option<String>>`, does it return only string?

Reply:
No, `reconciliation_err` it returns an object which I do not manage and know how to manage, but implements the to_string which allows me to evaluate if the error of a certain type.